Zombie/src/main/java/dk/gruppeseks/bodtrd/zombie/ZombieProcessor.java

```diff
@@ -111,4 +122,17 @@ public void notifyEntitiesRemoved(Entity entity)
     {
         throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
     }
+    
+    private void attackPlayer(World world, Entity zombie) 
```

Comment:
might aswell let the player be a parameter aswell given that you already get it in process